cinema/tests/test_movie_view_set.py

```diff
@@ -0,0 +1,130 @@
+from django.urls import reverse
+from rest_framework.test import APITestCase, APIClient
+from rest_framework import status
+from cinema.models import Movie, Genre, Actor
+from django.contrib.auth import get_user_model
+from rest_framework.authtoken.models import Token
+from io import BytesIO
+from PIL import Image
+
+User = get_user_model()
+
+def create_genre(name: str) -> Genre:
+    return Genre.objects.create(name=name)
+
+def create_actor(first_name: str, last_name: str) -> Actor:
+    return Actor.objects.create(first_name=first_name, last_name=last_name)
+
+def create_movie(title: str, genres=None, actors=None) -> Movie:
```

Comment:
This is a code quality issue. You have hardcoded the description and duration of a movie in the create_movie function. It would be better to pass these as parameters to the function, this will make your tests more flexible and representative of real use cases.